meteor/client/ui/Settings/ShowStyle/VariantSettings.tsx

```diff
@@ -79,42 +79,31 @@ export const ShowStyleVariantsSettings = withTranslation()(
 					return
 				}
 
-				_.forEach(newVariants, (entry) => {
-					MeteorCall.showstyles
-						.insertShowStyleVariantWithBlueprint(
-							entry.showStyleBaseId,
-							entry.blueprintConfig,
-							entry._id,
-							entry._rundownVersionHash,
-							entry.name
-						)
-						.catch((error) => {
-							NotificationCenter.push(
-								new Notification(
-									undefined,
-									NoticeLevel.WARNING,
-									t('Failed to import Variant ' + entry.name + '. Make sure it is not already imported.', {
-										errorMessage: error + '',
-									}),
-									'VariantSettings'
-								)
-							)
-						})
-				})
+				this.importVariantsFromArray(newVariants)
 			}
 			reader.readAsText(file)
 		}
+
+		importVariantsFromArray = (variants: ShowStyleVariant[]) => {
```

Comment:
This is a minor thing (but naming is important ;) )
Since your parameter is an array of ShowStyleVariants, it would be nice if they were named `showStyleVariants`

Reply:
Changed to use `showStyleVariant` everywhere!